server/csharp/TicketHub/Controllers/TicketController.cs

```diff
@@ -19,7 +20,8 @@ public class TicketController : ControllerBase
 {
     private readonly ILogger<TicketController> _logger;
     private readonly PostgresContext _dbContext;
-    private Dictionary<string, Func<ParameterExpression, Expression>> _ticketMapper;
+    private readonly Dictionary<string, Func<ParameterExpression, Expression>> _ticketMapper;

```

Comment:
Added the `readonly` marker on this private field, since it's set in the constructor, and nowhere else.